superset/commands/utils.py

```diff
@@ -62,6 +62,24 @@ def populate_owners(
     return owners
 
 
+def update_owner_list(
+    current_owners: list[User],
+    new_owners: list[int] | None,
+) -> list[User]:
+    """
+    Helper function for update commands, to properly handle the owners list.
+    Preserve the previous configuration unless included in the update payload.
+
+    :param current_owners: list of current owners
+    :param new_owners: list of new owners specified in the update payload
+    :returns: Final list of owners
+    """
+    owners_ids = (
+        [owner.id for owner in current_owners] if new_owners is None else new_owners
```

Comment:
can you add this small unit test that will basically verify the that `populate_owners` will be called with the proper values?

Reply:
Sure! I'm going to create a `tests/unit_tests/commands/test_utils.py` file. I found some other unit test files using `pytest` so I'll go with that.

Reply:
@hughhhh just added the unit tests 🙌 let me know if you have any feedback